hs-bindgen/tests/Misc.hs

```diff
@@ -36,6 +39,11 @@ goldenVsStringDiff_ name fp action = goldenTestSteps name correct action cmp upd
         | xss == yss = return Nothing
         | otherwise  = return $ Just $ ansidiff xss yss
 
+    normalise :: String -> String
+    normalise [] = []
+    normalise ('\r':'\n':r) = '\n' : normalise r
```

Comment:
again?

Reply:
I just cherry-picked the commit for updating the test-suite, I will drop it before merging